packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -617,6 +637,50 @@ describe('Testing authentication capability', () => {
           });
         });
 
+        it(`authentication.getUser should successfully get user profile with data residency info if dataResidency is provided by hosts`, (done) => {
```

Comment:
Could we also mention expect assertions at the top of each test? as it would help in failing appropriately incase all assertions are not called